tests/container_checker/test_container_checker.py

```diff
@@ -196,7 +173,7 @@ def get_expected_alerting_message(container_name):
     return expected_alerting_messages
 
 
-def test_container_checker(duthosts, enum_dut_feature_container, rand_selected_dut, tbinfo):
+def test_container_checker(duthosts, enum_dut_feature, enum_rand_one_per_hwsku_hostname, enum_rand_one_asic_index, tbinfo):
```

Comment:
is it possible to make fixture order in test argument and tuple member order as same for readability ?